examples/cartesian_reconstruction.py

```diff
@@ -15,15 +15,16 @@
 """
 
 # Package import
-from modopt.math.metrics import ssim
-from mri.operators import FFT
-from mri.optimizers import fista
-from mri.reconstruct.utils import generate_operators
+from mri.operators import FFT, WaveletN
 from mri.operators.utils import convert_mask_to_locations
-from pysap.data import get_sample_data
+from mri.reconstructors import SingleChannelReconstructor
 import pysap
+from pysap.data import get_sample_data
```

Comment:
Shouldn't we treat those as third-party? (minor comment)

Reply:
Even I dont know at this point . I feel it is package as pysap-mri cannot work without it and it is something maintained by us actively as a part of this project, pysap-mri is just a side extension

Reply:
As far as I understood the general idea behind this plugins thing is that the exemple should be called from pysap and not pysap-mri, as exemple we do not install pysap so I think that pysap cannot be considered as a third party import